addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/CommandInjectionScanRule.java

```diff
@@ -320,6 +327,10 @@ public String getReference() {
         return Constant.messages.getString(MESSAGE_PREFIX + "refs");
     }
 
+    public Map<String, String> getAlertTags() {
+        return ALERT_TAGS;
+    }
```

Comment:
Not a problem here, but what's out plan when a scan rule has multiple alerts which may have different applicable tags?

Reply:
Then its up to the alert to return the right ones :) Although if a rule returns alerts with radically different tags then maybe it should be split up into multiple rules?

Reply:
Same principle as with the other properties of the alert, they can be set when building the alert.

Reply:
Ok so this overridden (ultimately) method should just be the commons set. Which is fine.

Reply:
I dont think they have to be .. but probably best if they are...